lib/sisimai/bite/json/amazonses.rb

```diff
@@ -0,0 +1,255 @@
+module Sisimai::Bite::JSON
+  # Sisimai::Bite::JSON::AmazonSES parses a bounce object(JSON) which created
+  # by Amazon Simple Email Service.
+  # Methods in the module are called from only Sisimai::Message.
+  module AmazonSES
+    # Imported from p5-Sisimail/lib/Sisimai/Bite/JSON/AmazonSES.pm
+    class << self
+      require 'sisimai/bite/json'
+
+      # https://docs.aws.amazon.com/en_us/ses/latest/DeveloperGuide/notification-contents.html
+      BounceType = {
```

Comment:
__[rubocop]__
* Style/MutableConstant: Freeze mutable objects assigned to constants.

<!-- Comment by [SideCI](https://sideci.com) -->